[libvpx] Missing vcpkg_fixup_pkgconfing

libvpx install invalid pc files.

Found due to:
```
CMake Error in /mnt/vcpkg-ci/buildtrees/qtwebengine/x64-linux-dbg/CMakeFiles/CMakeTmp/CMakeLists.txt:
  Imported target "PkgConfig::VPX" includes non-existent path

    "/mnt/vcpkg-ci/packages/libvpx_x64-linux/debug/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.
```

--- a/ports/libvpx/portfile.cmake
+++ b/ports/libvpx/portfile.cmake
@@ -250,7 +250,7 @@ else()
 
         file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
         file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/lib/libvpx_g.a")
-    endif()    
+    endif()
 endif()
 
 vcpkg_fixup_pkgconfig()

--- a/ports/libvpx/portfile.cmake
+++ b/ports/libvpx/portfile.cmake
@@ -131,6 +131,16 @@ if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
             "${CURRENT_PACKAGES_DIR}/include"
         RENAME
             "vpx")
+    if (NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
+        set(LIBVPX_PREFIX "${CURRENT_INSTALLED_DIR}")
+        message("LIBVPX_PREFIX:${LIBVPX_PREFIX}")
+        configure_file("${CMAKE_CURRENT_LIST_DIR}/vpx.pc.in" "${CURRENT_PACKAGES_DIR}/lib/pkgconfig/vpx.pc" @ONLY)
+    endif()
+    
+    if (NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
+        set(LIBVPX_PREFIX "${CURRENT_INSTALLED_DIR}/debug")
+        configure_file("${CMAKE_CURRENT_LIST_DIR}/vpx.pc.in" "${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig/vpx.pc" @ONLY)   
+    endif()
 
 else()
 
@@ -240,11 +250,10 @@ else()
 
         file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
         file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/lib/libvpx_g.a")
-    endif()
-    
-    vcpkg_fixup_pkgconfig()
-    
+    endif()    
 endif()
+
+vcpkg_fixup_pkgconfig()
 
 if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
     set(LIBVPX_CONFIG_DEBUG ON)

--- a/ports/libvpx/portfile.cmake
+++ b/ports/libvpx/portfile.cmake
@@ -133,7 +133,6 @@ if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
             "vpx")
     if (NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
         set(LIBVPX_PREFIX "${CURRENT_INSTALLED_DIR}")
-        message("LIBVPX_PREFIX:${LIBVPX_PREFIX}")
         configure_file("${CMAKE_CURRENT_LIST_DIR}/vpx.pc.in" "${CURRENT_PACKAGES_DIR}/lib/pkgconfig/vpx.pc" @ONLY)
     endif()
     

--- a/ports/libvpx/portfile.cmake
+++ b/ports/libvpx/portfile.cmake
@@ -33,7 +33,7 @@ vcpkg_add_to_path(${NASM_EXE_PATH})
 
 if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
 
-    file(REMOVE_RECURSE ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET})
+    file(REMOVE_RECURSE "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}")
 
     if(VCPKG_CRT_LINKAGE STREQUAL static)
         set(LIBVPX_CRT_LINKAGE --enable-static-msvcrt)
@@ -71,7 +71,7 @@ if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
     endif()
 
     message(STATUS "Generating makefile")
-    file(MAKE_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET})
+    file(MAKE_DIRECTORY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}")
     vcpkg_execute_required_process(
         COMMAND
             ${BASH} --noprofile --norc
@@ -126,7 +126,7 @@ if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
     endif()
     file(
         INSTALL
-            ${LIBVPX_INCLUDE_DIR}
+            "${LIBVPX_INCLUDE_DIR}"
         DESTINATION
             "${CURRENT_PACKAGES_DIR}/include"
         RENAME
@@ -177,7 +177,7 @@ else()
 
     if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
         message(STATUS "Configuring libvpx for Release")
-        file(MAKE_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel)
+        file(MAKE_DIRECTORY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel")
         vcpkg_execute_required_process(
         COMMAND
             ${BASH} --noprofile --norc
@@ -210,7 +210,7 @@ else()
 
     if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
         message(STATUS "Configuring libvpx for Debug")
-        file(MAKE_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg)
+        file(MAKE_DIRECTORY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg")
         vcpkg_execute_required_process(
         COMMAND
             ${BASH} --noprofile --norc
@@ -238,9 +238,12 @@ else()
             LOGNAME install-${TARGET_TRIPLET}-dbg
         )
 
-        file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
-        file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/lib/libvpx_g.a)
+        file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+        file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/lib/libvpx_g.a")
     endif()
+    
+    vcpkg_fixup_pkgconfig()
+    
 endif()
 
 if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
@@ -249,6 +252,6 @@ else()
     set(LIBVPX_CONFIG_DEBUG OFF)
 endif()
 
-configure_file(${CMAKE_CURRENT_LIST_DIR}/unofficial-libvpx-config.cmake.in ${CURRENT_PACKAGES_DIR}/share/unofficial-libvpx/unofficial-libvpx-config.cmake @ONLY)
+configure_file("${CMAKE_CURRENT_LIST_DIR}/unofficial-libvpx-config.cmake.in" "${CURRENT_PACKAGES_DIR}/share/unofficial-libvpx/unofficial-libvpx-config.cmake" @ONLY)
 
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/libvpx/vcpkg.json
+++ b/ports/libvpx/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "libvpx",
   "version-semver": "1.10.0",
+  "port-version": 1,
   "description": "The reference software implementation for the video coding formats VP8 and VP9.",
   "homepage": "https://github.com/webmproject/libvpx"
 }

--- a/ports/libvpx/vpx.pc.in
+++ b/ports/libvpx/vpx.pc.in
@@ -9,5 +9,5 @@ Description: WebM Project VPx codec implementation
 Version: @LIBVPX_VERSION@
 Requires:
 Conflicts:
-Libs: -L"${libdir}" -lvpx -lm
+Libs: -L"${libdir}" -lvpx
 Cflags: -I"${includedir}"

--- a/ports/libvpx/vpx.pc.in
+++ b/ports/libvpx/vpx.pc.in
@@ -1,0 +1,13 @@
+prefix=@LIBVPX_PREFIX@
+# pkg-config file from libvpx v1.10.0
+exec_prefix=${prefix}
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: vpx
+Description: WebM Project VPx codec implementation
+Version: @LIBVPX_VERSION@
+Requires:
+Conflicts:
+Libs: -L"${libdir}" -lvpx -lm
+Cflags: -I"${includedir}"

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3746,7 +3746,7 @@
     },
     "libvpx": {
       "baseline": "1.10.0",
-      "port-version": 0
+      "port-version": 1
     },
     "libwandio": {
       "baseline": "4.2.1",

--- a/versions/l-/libvpx.json
+++ b/versions/l-/libvpx.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "ded52ca4a886b72794ced79a1945bd76d1f41967",
+      "git-tree": "55be5e09000631b1dacd6f86d174eb2cade91bf5",
       "version-semver": "1.10.0",
       "port-version": 1
     },

--- a/versions/l-/libvpx.json
+++ b/versions/l-/libvpx.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "c18a6892c9c95413757a0a1fe14e434c148e418c",
+      "git-tree": "81ba8315d736da6aa22c040561060fe9e4d0ff28",
       "version-semver": "1.10.0",
       "port-version": 1
     },

--- a/versions/l-/libvpx.json
+++ b/versions/l-/libvpx.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "81ba8315d736da6aa22c040561060fe9e4d0ff28",
+      "git-tree": "ded52ca4a886b72794ced79a1945bd76d1f41967",
       "version-semver": "1.10.0",
       "port-version": 1
     },

--- a/versions/l-/libvpx.json
+++ b/versions/l-/libvpx.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "c18a6892c9c95413757a0a1fe14e434c148e418c",
+      "version-semver": "1.10.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "ba9e1f9d346412c76f17d9fda5a3fbd5b5555afa",
       "version-semver": "1.10.0",
       "port-version": 0

